### ISSUE TITLE
404 on examples code

### ISSUE THREAD

**Issue by OKNoah** [react-dnd/react-dnd#990]
Clicking browse the source leaves me with 404 on some page, like these:

http://react-dnd.github.io/react-dnd/examples-drag-around-naive.html -> [Goes here](https://github.com/react-dnd/react-dnd/tree/master/examples/05%20Customize/Drop%20Effects)
http://react-dnd.github.io/react-dnd/examples-customize-drop-effects.html -> [Goes here](https://github.com/react-dnd/react-dnd/tree/master/examples/05%20Customize/Drop%20Effects)

**Comment by OKNoah** [comment]
I think they lack the `/packages` directory because you're now using `lerna`.

**Comment by 29er** [comment]
so where can we find the examples now ?

**Comment by darthtrevino** [comment]
Yes, I pushed the docs into the packages/ structure yesterday. I'll have the docs fixed in a moment.
